php.ini

```diff
@@ -45,3 +40,7 @@ session.gc_probability = 1
 ;
 opcache.validate_timestamps = 0
 opcache.blacklist_filename="${MAGENTO_CLOUD_APP_DIR}/op-exclude.txt"
+opcache.memory_consumption=512
```

Comment:
opcache.memory_consumption=256  << this should be 64 on integration
opcache.max_accelerated_files=16229

Reply:
Shouldn't opcache.max_accelerated_file be >= # of Magento files?

Reply:
`opcache.memory_consumption` is 128 by default https://www.php.net/manual/en/opcache.configuration.php#ini.opcache.memory-consumption

Reply:
> Shouldn't opcache.max_accelerated_file be >= # of Magento files
No, only used files

Do we have possibility to track target branch on build service? We should generate php.ini memory_consumption based on type of environment

Reply:
Also you can use https://github.com/rlerdorf/opcache-status to get statistics